afs-core/src/main/java/com/powsybl/afs/AppFileSystemTool.java

```diff
@@ -191,7 +203,13 @@ private void runArchive(CommandLine line, ToolRunningContext context) {
             Path dir = context.getFileSystem().getPath(line.getOptionValue(DIR));
             boolean mustZip = line.hasOption(ZIP);
             boolean archiveDependencies = line.hasOption(DEPENDENCIES);
-            fs.getRootFolder().archive(dir, mustZip, archiveDependencies);
+            boolean deleteResult = line.hasOption(DELETE_RESULT_OPTNAME);
+            Map<String, List<String>> outputBlackList = new HashMap<>();
+            if (deleteResult) {
+                List<ProjectFileExtension> services = PROJECT_FILE_EXECUTION.getServices();
```

Comment:
Not sure this variable is needed: use the return of `PROJECT_FILE_EXECUTION.getServices()` directly